src/file_io.jl

```diff
@@ -105,7 +105,8 @@ end
 
 Creates a JLSOFile with the specified data and kwargs and writes it back to the io.
 """
-save(io::IO, data; kwargs...) = write(io, JLSOFile(data; kwargs...))
+save(io::IO, data::JLSOFile) = write(io, data)
+save(io::IO, data; kwargs...) = save(io, JLSOFile(data; kwargs...))
```

Comment:
since this now delegates to the new method above, this new method gets exercised by the existing tests

Reply:
Yeah, that seems fine.